pandas/core/shared_docs.py

```diff
@@ -94,7 +94,10 @@
     If ``by`` is a function, it's called on each value of the object's
     index. If a dict or Series is passed, the Series or dict VALUES
     will be used to determine the groups (the Series' values are first
-    aligned; see ``.align()`` method). If an ndarray is passed, the
+    aligned; see ``.align()`` method). If a list (of acceptable items.
+    See the `groupby user guide
+    <https://pandas.pydata.org/pandas-docs/stable/user_guide/groupby.html#splitting-an-object-into-groups>`)
```

Comment:
Could you move ``See the `groupby user guide
    <https://pandas.pydata.org/pandas-docs/stable/user_guide/groupby.html#splitting-an-object-into-groups>`)`` after passed? Not sure if we need the acceptable items part

Reply:
Thank you for the input. I have updated doc string.